packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/simple-duration.tsx

```diff
@@ -15,12 +30,13 @@ const SimpleDuration = observer(
         getDurationFromUnit,
         number_input_props,
         shared_input_props,
-    }) => {
+        simple_duration_unit,
+    }: TSimpleDuration) => {
         const { ui } = useStore();
-        const { current_focus, setCurrentFocus, simple_duration_unit } = ui;
```

Comment:
Parent component was passing `simple_duration_unit` as a prop, so I removed extraction from ui store

Reply:
it's better to still use `simple_duration_unit` from ui-store as a fallback since the parent doesn't always receive `simple_duration_unit`.
like so:
simple_duration_unit || ui.simple_duration_unit